src/docs/demos/modal/modal-demo.controller.js

```diff
@@ -1,13 +1,9 @@
 class controller {
-  constructor ($ngbsModal) {
-    this.$ngbsModal = $ngbsModal;
+  constructor($log, $ngbsModal) {
+    Object.assign(this, { $log, $ngbsModal });
```

Comment:
Nice touch, but i'm not sure i'm willing to say bye bye to IE users.

There's a nice [babel plugin](https://babeljs.io/docs/plugins/transform-object-assign/), and also a [polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill). What do you think?